### DISCUSSION TITLE
경단 프로젝트 회고

### DISCUSSION THREAD

**Discussion by stopmin** [BDD-CLUB/challenge-every-week-posting#601] [5. 회고]
안녕하세요 선배님들 :D

제가 2주나.. 글을 빠지다니!!! 미안합니다!! 
이번에 경단 프로젝트를 준비하고 있었고, 카테캠도 경고를 5번이나 먹어버렸습니다하...!!ㅎㅎ 
이번에 프로젝트 진행하면서 몸도 많이 안좋아지고, 우는 날도 있었고 웃는 날도 있었는데 막상 디지털 경진대회 제출하고 나니 마음이 한결 가벼워지네요!
ㅎㅎ 혹시나 웹 보고 싶으신 분은 아래 링크로 한번 구경해보셔도 될거같아요~! 창피하지만 올려봅니당 

https://gyeongdan.vercel.app/newsletter

글구 이번에 Next랑 타입스크립트도 해봤는데.. 프론트 정말 어렵더라고요 ㅠㅠ 하핫 재밌었습니다!!

그리고 경단 회고 글을 좀 남겨봤습니당 ㅎㅎ
[블로그 글 바로가기](https://stopmin.tistory.com/entry/%EA%B2%BD%EB%8B%A8%EA%B2%BD%EC%A0%9C%EB%A5%BC-%EB%8B%A8%EC%88%9C%ED%95%98%EA%B2%8C-%ED%94%84%EB%A1%9C%EC%A0%9D%ED%8A%B8-SW%EC%A4%91%EC%8B%AC%EB%8C%80%ED%95%99-%EB%94%94%EC%A7%80%ED%84%B8-%EA%B2%BD%EC%A7%84%EB%8C%80%ED%9A%8C-%ED%9A%8C%EA%B3%A0)

다들 좋은 저녁 되세요😄

**Comment by shkisme** [comment]
ㄷㄷ 무서울 정도로 잘하시네요

**Reply by stopmin** [reply]
아닙니당!!!

**Comment by pykido** [comment]
경단 화이팅~!

**Reply by stopmin** [reply]
경단 빠이팅~~

**Comment by 1jeongg** [comment]
헉 진짜 잘 만드신 것 같아요! UI도 너무 예쁘고 유용한 서비스 같습니다..! 대회에서도 좋은 결과 있으시길 바랄게요 :)

근데 마을 탭에 지도에 마커 클릭하면 404 에러 뜨는 것 같아요!

**Reply by stopmin** [reply]
맞습니당~~!! 아직 사실 안붙였습니다 헤헷..!! 
감사합니당 :D

**Reply by stopmin** [reply]
인제 될겁니다~!
